### ISSUE TITLE
Screen window is blocked by control panels

### ISSUE THREAD

**Issue by lauracumming** [ocadotechnology/rapid-router#1118]
On the road side of Rapid Router levels, the controls seem to sit on a green background which reduces the size of the viewing pane for the actual level, meaning that if a user increases the size of the map it is cut off at the sides.

![Screenshot 2020-09-08 at 08.31.39.png](https://images.zenhubusercontent.com/5da58aa016098c0001fb85d1/084ca9dc-0bb9-4aee-bf58-4fa2dd0060d9)

**Comment by faucomte97** [comment]
No longer relevant.
